### PR TITLE
Recommended: V0.1.0 pre

### PR DESCRIPTION
**Pull request by rwaldron**
While investigating the issues in this thread: https://forums.tessel.io/t/typeerror-cannot-read-property-pin-of-undefined/2815/3 I made use of some available cycles to get this module updated and tested for Tessel 2 use. I recommend reviewing the changes by commit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,43 +44,39 @@ a body is detected and when a body exits
...
 
-pir.on('ready', function (pir) {
+pir.on('ready', pir => {
   console.log('Ready and waiting...');
-  pir.on('movement', function (time) {
-    console.log('Something moved! Time ' + time);
+  pir.on('movement:start', time => {
+    console.log(`Something moved! Time ${time}`);
   });
-  pir.on('stillness', function (time) {
```
Comment (rwaldron) [inline README.md]: These are still supported.

File: index.js [hunk 2]
```diff
@@ -1,56 +1,52 @@
...
 
-// Functions
-// Read the state of the pin
-PIR.prototype.read = function (callback) {
-  if(callback) {
-    callback(this.hardware.read());
+  static get PIR() {
+    return PIR;
   }
-  return this.hardware.read();
```
Comment (rwaldron) [inline index.js]: Synchronous reads aren't supportable on Tessel 2, and this isn't really useful for the sensor itself (the `movement` property and related events should be sufficient for all use cases)

File: test/apiTest.js [hunk 3]
```diff
@@ -1,42 +0,0 @@
...
-  }),
-  
-  test('Read function', function (t) {
-    pir.read(function (data) {
-      t.ok((data === 0) || (data === 1), 'Read should return 0 or 1 (callback value).');
-    });
-    t.ok((pir.read() === 0) || (pir.read() === 1), 'Read should return 0 or 1 (returned value).');
-  })
-  
-  ]);
```
Comment (rwaldron) [inline test/apiTest.js]: Replaced by unit tests

**Comment by Frijol** [comment]
Whoa, thanks for taking a look! I will try to review this week

**Comment by Frijol** [comment]
Amazing. Thank you very much!

**Pull request merged**